src/main/java/pl/touk/sputnik/connector/stash/json/Anchor.java

```diff
@@ -16,8 +16,8 @@
 @Builder
 public class Anchor {
     public int line;
-    public String lineType = "CONTEXT";
-    public String fileType = "TO";
+    public String lineType;
+    public String fileType;
```

Comment:
\[Checkstyle\] ERROR: Variable 'fileType' must be private and have accessor methods\.